code/controllers/subsystem/dcs.dm

```diff
@@ -0,0 +1,32 @@
+SUBSYSTEM_DEF(dcs)
+	name = "Datum Component System"
+	flags = SS_NO_INIT | SS_NO_FIRE
+
+	var/list/comp_lookup = list()	// A signal:list(components) assoc list
```

Comment:
You need to add a /Recover() override that preserves this

Reply:
done